packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -242,10 +257,14 @@ const BlockInspector = ( { showNoBlockSelectedMessage = true } ) => {
 };
 
 const BlockInspectorSingleBlock = ( { clientId, blockName } ) => {
+	const tabSettings = useSelect( ( select ) => {
+		return select( blockEditorStore ).getSettings()
+			.__experimentalBlockInspectorTabs;
+	}, [] );
```

Comment:
I wonder if this and `getShowTabs` could be folded into the `useInspectorControlsTabs` hook.